data/VT/incentive_relationships.json

```diff
@@ -1,17 +1,25 @@
 {
   "prerequisites": {
```

Comment:
See the "relationships" tab I added to [the spreadsheet](https://docs.google.com/spreadsheets/d/1mAHr_CQWwMpccOK6E1i09334GcojxGAN8Z4n-jO6cQg/edit) to understand everything that's going on here.

This makes me think two things:
- Spreadsheets are terrible for capturing these complex relationships. (More ammo for the CMS argument?)
- Maybe we should allow comments in these JSON files? This is pretty inscrutable by itself.

Reply:
JSON5 could be a good drop-in that allows comments?